client/src/views/EventsView.vue

```diff
@@ -111,4 +111,11 @@ function openEditModal(id: number) {
   padding-bottom: 0px;
   margin-top: -19px;
 }
+
+@media (min-width: 960px) {
+  .container {
+    margin-left: 15px;
+    margin-right: 15px;
+  }
+}
```

Comment:
I dont think we would need this if we change to use a v-container. Let me know if it works or not